config/ember-try.js

```diff
@@ -2,138 +2,95 @@
 
 const getChannelURL = require('ember-source-channel-url');
 
-module.exports = async function() {
+module.exports = async function () {
   return {
     useYarn: true,
     scenarios: [
       {
-        name: 'ember-lts-2.8',
-        bower: {
-          dependencies: {
-            ember: 'components/ember#lts-2-8',
-          },
-          resolutions: {
-            ember: 'lts-2-8',
-          },
-        },
+        name: 'ember-lts-3.8',
         npm: {
           devDependencies: {
-            'ember-source': null,
-            'ember-factory-for-polyfill': '1.3.1'
+            'ember-source': '~3.8.0',
+            'ember-data': '3.11.5',
```

Comment:
Any reason to freeze ember-data here?

Reply:
Pretty sure it blows up if we don't

Reply:
I think it was something with the docs, but I forget. This was added here per @runspired's suggestion, since it allowed us to bump ember-data in the rest of the package.